Things.md

```diff
@@ -0,0 +1,126 @@
+This is a general guide about Things for all FullScreenShenanigans projects. This guide will show you how in-game Things are stored and manipulated. Everything you see in the game (trees, blocks, the player, etc.) is a Thing.  
+The Thing class is subclassed by a new class for everything (Tree class, Block class, Player class, etc.). These Things can be manipulated to change their properties, placements, etc.    
+  
+### Table of Contents  
+1. [GroupHoldr](#groupholdr)
+    1. [Getting Groups](#getting-groups)
+2. [Manipulating Things](#manipulating-things)
+    1. [Shifting Things](#shifting-things)
+    2. [Opacity](#opacity)
+
+# GroupHoldr  
+
+GroupHoldr is a way to store similarly typed information, such as Things, based on a shared property. Each Thing has a `groupType` string property that determines what group it's considered to be in. Some examples from `FSM` (FullScreenmario) are:
+
+* Text
+* Character
+* Solid
+* Scenery 
+
+GroupHoldr contains an Array for each of the groups; each Array contains all the Things of that type currently in the game. Things are added to their respective group when added to the game, and removed when they die. For example, in `FSM`, solid objects like floors and blocks would be stored in the **Solid** group. 
+
+### Getting Groups  
+The groups are accessible both by static name and via passing in a String:
+
+```javascript
+// Returns the Solid group
+this.getGroup("Solid");
+
+// Returns the first Solid
+this.getGroup("Solid")[0];
+this.getSolid(0);
+```
+
+More can be read on GroupHoldr on its [Readme](https://github.com/FullScreenShenanigans/GroupHoldr/blob/master/README.md).
```

Comment:
You should also put a code sample storing `thing` and `otherThing` as vars, since you use them later on. People are going to run the code samples and get confused about what those two names mean.